From 2d1f612ca497d8aea00b35da4a825f72312aa064 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Tue, 2 Apr 2013 15:00:37 +0800
Subject: [PATCH 15/95] ENGR00255881 fix Camera preview freezes for some USB
 Camera.

some usb cameras take long time from closing to opening and set
parameter then dequeue a frame.
so, enlarge the taking picture waiting timeout value.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 services/camera/libcameraservice/api1/client2/CaptureSequencer.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/camera/libcameraservice/api1/client2/CaptureSequencer.h b/services/camera/libcameraservice/api1/client2/CaptureSequencer.h
index 10252fb..8019dd9 100644
--- a/services/camera/libcameraservice/api1/client2/CaptureSequencer.h
+++ b/services/camera/libcameraservice/api1/client2/CaptureSequencer.h
@@ -110,7 +110,7 @@ class CaptureSequencer:
     static const nsecs_t kWaitDuration = 100000000; // 100 ms
     static const int kMaxTimeoutsForPrecaptureStart = 10; // 1 sec
     static const int kMaxTimeoutsForPrecaptureEnd = 20;  // 2 sec
-    static const int kMaxTimeoutsForCaptureEnd    = 40;  // 4 sec
+    static const int kMaxTimeoutsForCaptureEnd    = 100;  // 10 sec
 
     wp<Camera2Client> mClient;
     wp<ZslProcessorInterface> mZslProcessor;
-- 
1.8.0

